### PR TITLE
feat: delete sessionId on 5xx errors

### PR DESCRIPTION
**Pull request by rbrazhnyk**

### CODE REVIEW

File: src/services/affinidi/errors.ts [hunk 1]
```diff
@@ -65,6 +66,10 @@ export function handleServiceError(
   serviceErrorMessageHandler?: (response: any) => string | null,
 ): never {
   if (error instanceof AxiosError && error.response) {
+    if (error.response.status >= 500) {
+      credentialsVault.clear()
```
Comment (aeffinidi) [inline src/services/affinidi/errors.ts:70]: dumb question:  don't we need to return after the `credentials` are cleared ?

Comment (rbrazhnyk) [reply src/services/affinidi/errors.ts:70]: After moving to BFF, `sessionId` became the only thing that is stored in file ([see service for the reference](https://github.com/affinidi/affinidi-cli/blob/main/src/services/credentials-vault.ts#L3)).

Comment (rbrazhnyk) [reply src/services/affinidi/errors.ts:70]: Throwing `CLIError` in the end of the block returns a nice message to the user.

**Review by pulkitb2 — APPROVED** [review]

**Pull request merged**